resources/assets/styles/tailwind.js

```diff
@@ -47,7 +53,47 @@ module.exports = {
       '5': '32px',
       '6': '48px',
     },
+    gutenberg: (theme) => ({
+      colors: {
```

Comment:
So these colors, are used here to generate classes using them?
https://github.com/generoi/tailwindcss-gutenberg/blob/master/plugins/colors.js

Reply:
Yes

Reply:
At first I just used tailwinds own classes but that sort of removes the option to have shades of colors.